src/App.js

```diff
@@ -1,12 +1,66 @@
-import React from 'react';
+import React, {useEffect, useState} from "react";
+import axios from 'axios';
 import './App.css';
+import PokemonCards from "./components/pokemon/PokemonCards";
+import Button from "./components/button/Button";
 
-function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+
+// Het resultaat opslaan in state.
+// Een console log maken van het resultaat
+// Bekijk wat je terug krijgt van de request in de dev tools
+
+function  App() {
+    const [pokemon, setPokemon] = useState  ([])
```

Comment:
Zodra de setter is aangeroepen is `pokemon` niet meer van het type array. Ik zou daarom de default waarde `undefined` maken en dan kan je check onderin ook iets simpeler
```suggestion
    const [pokemon, setPokemon] = useState(undefined);
```